### PR TITLE
feat: Created signup page for individuals and organizations

### LINKED ISSUE #19: SignUp UI Implementation
## Description:
- Implement SignUp UI using NextUI or Material UI (mui) and Tailwind CSS according to the following design
- On Clicking the SignUp button on HomePage,a popup will open where user can select whether to signup as an individual or organization
- On Clicking SignUp As an Individual/Sign Up as an Organization, conditionally render respective forms
- Make sure that the UI is fully responsive

## Design:
 
SignUP PopUP | 
SignUp-Organisation |
SignUp-Individual

### PR DESCRIPTION
**Pull request by MahendraDani**
Fixes #19 

## Outcomes
1. Created login page for both individuals and organizations.
2. The route for individual signup is `http://localhost:3000/signup/individual`.
3. The route for organizations signup is `http:localhost:3000/signup/organization`.
4. Fixed imports of images in `home page`.
5. All the pages are **fully responsive** for seamless user experience.
6. Added `graph` image in `homepage`.
## Video/Screenshots

https://github.com/SrijanShovit/carbonops-v2/assets/116940083/31b4b69b-7639-47fe-b499-509b2abeeabb

### CODE REVIEW

**Comment by MahendraDani** [comment]
@shreya024 Please review the changes.
If there are any changes please let me know!
Thanks!

**Comment by shreya024** [comment]
> @shreya024 Please review the changes. If there are any changes please let me know! Thanks!

Hey @MahendraDani Please remove all warnings in your console,so that vercel can deploy successfully

**Comment by MahendraDani** [comment]
@shreya024 How should I clear the vercel test?
Do I need to commit again ?
Please help me out!

File: app/signup/individual/page.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import cakeIcon from "../../../public/icons/cake.png";
+const IndividualSignup = () => {
+  return (
+    <>
+      <div className="text-lg w-full h-screen flex flex-col justify-center items-center my-16 md:my-0">
+        <div className="w-[90%] md:w-[70%] mx-auto bg-[#12323f]">
+          <div className="flex flex-col md:flex-row justify-between items-center p-4">
+            <div className="w-full flex flex-col justify-between items-center gap-2 border-b-2 border-slate-400 md:border-b-0 md:border-r-2 pb-4">
+              <div>
+                <Image src={welcomeImage} height={500} width={500} />
```
Comment (shreya024) [inline app/signup/individual/page.js:15]: Add alt tags to all <Image> tags

**Review by shreya024 — CHANGES_REQUESTED** [review]

File: app/signup/individual/page.js (hunk 1 above)
Comment (MahendraDani) [reply app/signup/individual/page.js:15]: Will try to resolve all warnings by tonight!
Thanks!

**Comment by shreya024** [comment]
> @shreya024 How should I clear the vercel test? Do I need to commit again ? Please help me out!

When you run the app on localhost,right-click and click on Inspect ->go to console->you will see a bunch of warnings like this
image
just add the alt tags and any other changes as is shown in the console and push again

**Comment by MahendraDani** [comment]
> > @shreya024 How should I clear the vercel test? Do I need to commit again ? Please help me out!
> 
> When you run the app on localhost,right-click and click on Inspect ->go to console->you will see a bunch of warnings like this image just add the alt tags and any other changes as is shown in the console and push again

@shreya024 Do I need to clear the yellow warnings as well? Because the dimensions of images are set which suits bests for the UI.

**Comment by shreya024** [comment]
> > > @shreya024 How should I clear the vercel test? Do I need to commit again ? Please help me out!
> > 
> > 
> > When you run the app on localhost,right-click and click on Inspect ->go to console->you will see a bunch of warnings like this image just add the alt tags and any other changes as is shown in the console and push again
> 
> @shreya024 Do I need to clear the yellow warnings as well? Because the dimensions of images are set which suits bests for the UI.

Push once with all red warnings cleared.If vercel fails again,you may need to work on the yellow warnings as well

**Comment by MahendraDani** [comment]
@shreya024 I have resolved all the red warnings appearing in the console, still the Vercel test is failing.
Would you please tell me what should I do now?

**Comment by shreya024** [comment]
> @shreya024 I have resolved all the red warnings appearing in the console, still the Vercel test is failing. Would you please tell me what should I do now?

Try replacing ' in content with `&apos` 
For example: industry's will become industry `&apos` s

**Comment by MahendraDani** [comment]
@shreya024 
What if I removed all the ' from the content?
Will it be fine for now?

Because once the pages are set up we can optimize the SEO using the alt tags later.
Right?

**Comment by shreya024** [comment]
> @shreya024 
> What if I removed all the ' from the content?
> Will it be fine for now?
> 
> Because once the pages are set up we can optimize the SEO using the alt tags later.
> Right?

Yeah sure,u can remove them for now

**Comment by MahendraDani** [comment]
> > @shreya024
> > What if I removed all the ' from the content?
> > Will it be fine for now?
> > Because once the pages are set up we can optimize the SEO using the alt tags later.
> > Right?
> 
> Yeah sure,u can remove them for now

Removing them for now!

**Comment by MahendraDani** [comment]
@shreya024 I have changed all `'` with `&apos;` in the codebase, still the vercel test is failing.
Now I can't think of any reason why it is failing?

**Comment by SrijanShovit** [comment]
@shreya024 
[21:42:07.464] Running build in Washington, D.C., USA (East) – iad1
[21:42:07.523] Cloning github.com/SrijanShovit/carbonops-v2 (Branch: Mahendra/signup_page_branch, Commit: 2544b06)
[21:42:07.821] Cloning completed: 298.307ms
[21:42:10.710] Restored build cache
[21:42:10.739] Running "vercel build"
[21:42:11.254] Vercel CLI 29.3.6
[21:42:11.528] Installing dependencies...
[21:42:13.213] 
[21:42:13.214] added 1 package in 1s
[21:42:13.214] 
[21:42:13.214] 136 packages are looking for funding
[21:42:13.214]   run `npm fund` for details
[21:42:13.225] Detected Next.js version: 13.4.2
[21:42:13.227] Detected `package-lock.json` generated by npm 7+...
[21:42:13.228] Running "npm run build"
[21:42:13.538] 
[21:42:13.538] > carbonops-v2@0.1.0 build
[21:42:13.538] > next build
[21:42:13.539] 
[21:42:14.081] - info Creating an optimized production build...
[21:42:22.797] Failed to compile.
[21:42:22.797] 
[21:42:22.797] ./components/homepage/Services.js
[21:42:22.797] Module not found: Can't resolve '../../public/images/homepage/interview.png'
[21:42:22.798] 
[21:42:22.798] https://nextjs.org/docs/messages/module-not-found
[21:42:22.798] 
[21:42:22.798] Import trace for requested module:
[21:42:22.798] ./app/page.js
[21:42:22.798] 
[21:42:22.798] 
[21:42:22.799] > Build failed because of webpack errors
[21:42:22.839] Error: Command "npm run build" exited with 1
[21:42:23.031] BUILD_UTILS_SPAWN_1: Command "npm run build" exited with 1

Problem in deployment

**Comment by shreya024** [comment]
@MahendraDani this is the error log. Kindly go through it and fix the bugs

**Comment by MahendraDani** [comment]
@shreya024 yes mam!

**Comment by MahendraDani** [comment]
> @shreya024 [21:42:07.464] Running build in Washington, D.C., USA (East) – iad1 [21:42:07.523] Cloning github.com/SrijanShovit/carbonops-v2 (Branch: Mahendra/signup_page_branch, Commit: [2544b06](https://github.com/SrijanShovit/carbonops-v2/commit/2544b06d742369040af95bfa8c65cd620bb5b925)) [21:42:07.821] Cloning completed: 298.307ms [21:42:10.710] Restored build cache [21:42:10.739] Running "vercel build" [21:42:11.254] Vercel CLI 29.3.6 [21:42:11.528] Installing dependencies... [21:42:13.213] [21:42:13.214] added 1 package in 1s [21:42:13.214] [21:42:13.214] 136 packages are looking for funding [21:42:13.214] run `npm fund` for details [21:42:13.225] Detected Next.js version: 13.4.2 [21:42:13.227] Detected `package-lock.json` generated by npm 7+... [21:42:13.228] Running "npm run build" [21:42:13.538] [21:42:13.538] > carbonops-v2@0.1.0 build [21:42:13.538] > next build [21:42:13.539] [21:42:14.081] - info Creating an optimized production build... [21:42:22.797] Failed to compile. [21:42:22.797] [21:42:22.797] ./components/homepage/Services.js [21:42:22.797] Module not found: Can't resolve '../../public/images/homepage/interview.png' [21:42:22.798] [21:42:22.798] https://nextjs.org/docs/messages/module-not-found [21:42:22.798] [21:42:22.798] Import trace for requested module: [21:42:22.798] ./app/page.js [21:42:22.798] [21:42:22.798] [21:42:22.799] > Build failed because of webpack errors [21:42:22.839] Error: Command "npm run build" exited with 1 [21:42:23.031] BUILD_UTILS_SPAWN_1: Command "npm run build" exited with 1
> 
> Problem in deployment

@SrijanShovit @shreya024 
I fixed the bugs from this error log.
I request you to please solve this error for me.
I am trying to pass the vercel test but I am unable to do so. I don't know why this is happening but I am trying my best to fix this. I have worked hard to implement the `Signup Pages` for the website.
Please help me out!

File: components/homepage/Services.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import Image from "next/image";
+import serviceImage1 from "../../public/images/homepage/service-1.png";
+import serviceImage2 from "../../public/images/homepage/interview.png";
```
Comment (shreya024) [inline components/homepage/Services.js:3]: Change `import serviceImage2 from "../../public/images/homepage/interview.png"` to `import serviceImage2 from "../../public/images/homepage/Interview.png"

Comment (MahendraDani) [reply components/homepage/Services.js:3]: But the file name of the image is saved as `interview.png` in the file directory.

**Comment by shreya024** [comment]
> > @shreya024 [21:42:07.464] Running build in Washington, D.C., USA (East) – iad1 [21:42:07.523] Cloning github.com/SrijanShovit/carbonops-v2 (Branch: Mahendra/signup_page_branch, Commit: [2544b06](https://github.com/SrijanShovit/carbonops-v2/commit/2544b06d742369040af95bfa8c65cd620bb5b925)) [21:42:07.821] Cloning completed: 298.307ms [21:42:10.710] Restored build cache [21:42:10.739] Running "vercel build" [21:42:11.254] Vercel CLI 29.3.6 [21:42:11.528] Installing dependencies... [21:42:13.213] [21:42:13.214] added 1 package in 1s [21:42:13.214] [21:42:13.214] 136 packages are looking for funding [21:42:13.214] run `npm fund` for details [21:42:13.225] Detected Next.js version: 13.4.2 [21:42:13.227] Detected `package-lock.json` generated by npm 7+... [21:42:13.228] Running "npm run build" [21:42:13.538] [21:42:13.538] > carbonops-v2@0.1.0 build [21:42:13.538] > next build [21:42:13.539] [21:42:14.081] - info Creating an optimized production build... [21:42:22.797] Failed to compile. [21:42:22.797] [21:42:22.797] ./components/homepage/Services.js [21:42:22.797] Module not found: Can't resolve '../../public/images/homepage/interview.png' [21:42:22.798] [21:42:22.798] https://nextjs.org/docs/messages/module-not-found [21:42:22.798] [21:42:22.798] Import trace for requested module: [21:42:22.798] ./app/page.js [21:42:22.798] [21:42:22.798] [21:42:22.799] > Build failed because of webpack errors [21:42:22.839] Error: Command "npm run build" exited with 1 [21:42:23.031] BUILD_UTILS_SPAWN_1: Command "npm run build" exited with 1
> > Problem in deployment
> 
> @SrijanShovit @shreya024 I fixed the bugs from this error log. I request you to please solve this error for me. I am trying to pass the vercel test but I am unable to do so. I don't know why this is happening but I am trying my best to fix this. I have worked hard to implement the `Signup Pages` for the website. Please help me out!

Hey @MahendraDani Don't worry.Your PR will be merged.and we are with you to provide all the help you need to solve these kind of errors.Consider this as a part of the learning process.

**Comment by MahendraDani** [comment]
@shreya024 I have checked on my local development server. All the images are properly displayed and there is no error in neither the terminal nor the console.

Yesterday I changed the file name of `serviceImage2` as `interview.png` which makes the import path correct. So there should not be any error in tha file path?

**Comment by shreya024** [comment]
> @shreya024 I have checked on my local development server. All the images are properly displayed and there is no error in neither the terminal nor the console.
> 
> Yesterday I changed the file name of `serviceImage2` as `interview.png` which makes the import path correct. So there should not be any error in tha file path?

in the images folder it is saved as Interview.png right?...that's the vercel error here it cannot find 'interview.png'

**Comment by MahendraDani** [comment]
@shreya024 Now I have changed the path to the one mentioned by you.
I think there is some issue in vercel deployments that is failing the tests, because if there were any errors in my code then it would have shown in my development server but when I am checking on my development server I see no errors and then I push the commit but after that the vercel test fails here, which is why I am not able understand what the error is exactly?!!

**Comment by shreya024** [comment]
@MahendraDani In ./components/homepage/Hero.js change `../../public/images/homepage/hero.png` to `../../public/images/homepage/Hero.png`
In ./app/signup/page.js
In lines 31 and 62 change ' to &apos;

**Comment by shreya024** [comment]
> @MahendraDani In ./components/homepage/Hero.js change `../../public/images/homepage/hero.png` to `../../public/images/homepage/Hero.png` In ./app/signup/page.js In lines 31 and 62 change ' to `&apos;`

**Comment by MahendraDani** [comment]
@shreya024 Mam all checks are passed this time.
I can't thank you enough for helping me out!

I am extremely sorry for the 
inconvenience cause during the process.

I will try my best to not make such mistakes again.
Thanks a lot!

**Review by shreya024 — APPROVED** [review]
Good job 👏 @MahendraDani

**Comment by MahendraDani** [comment]
@shreya024 Thanks mam!
Thanks a lot for providing help throughout the process!

**Pull request merged**